pset-functions.js

```diff
@@ -30,7 +30,18 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+const blackJack = (playerCardScore, dealerCardScore) => {
+    if (playerCardScore > 21 && dealerCardScore > 21){
+        return 0
+    }else if (dealerCardScore > 21){
+        return playerCardScore
+    } else if (playerCardScore > 21 ){
+        return dealerCardScore
+    } else {
+        return Math.max(playerCardScore, dealerCardScore)
```

Comment:
Sammi, this is a great use of if/else statements; you capture every possible scenario and also take advantage of the built in Math methods. Very well done on this one.